apps/web/src/app/(dashboard)/documents/[id]/edit-document.tsx

```diff
@@ -47,9 +49,35 @@ export const EditDocumentForm = ({
 }: EditDocumentFormProps) => {
   const { toast } = useToast();
   const router = useRouter();
+  const { data: session } = useSession();
```

Comment:
`useSession` looks to be redundant in this scenario since we pass `user` down as a prop

See a few lines up and change

```
user: _user,
```

to

```
user
```

Reply:
done... making a new commit